src/scss/components/_dropdown.scss

```diff
@@ -0,0 +1,36 @@
+.dropdown-toggle {
+  position: relative;
+}
+
+.dropdown {
+  display: none;
+  background-color: #ffffff;
+  list-style-type: none;
```

Comment:
cái này move vào _reset.scss style cho thẻ ul nhé. để nó apply cho toàn bộ các tag ul dùng sau này, e đỡ phải style đi style lại cái thuộc tính này